### PR TITLE
feat: implement `TimezonesCollapseButton` component

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- Implemented `TimezoneCollapsedButton` with related options & an instance event.
  - `showTimezoneCollapseButton`, `timezonesCollapsed` options
  - `clickTimezoneCollapseBtn` event

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/src/css/timegrid/timeColumn.css [hunk 1]
```diff
@@ -55,3 +55,38 @@
...
   transform: translateY(-50%);
 }
+
+.timezone-labels-slot {
+  display: table;
+  table-layout: fixed;
+  position: absolute;
+  height: 40px;
+  border-bottom: 1px solid #e9e9e9;
+  background-color: white;
```
Comment (jwlee1108) [inline apps/calendar/src/css/timegrid/timeColumn.css:65]: 별 차이는 없지만 색상 키워드 대신 HEX로 하는건 어떨까요?

**Review by jwlee1108 — COMMENTED** [review]
리뷰 완료합니다.

File: apps/calendar/src/css/timegrid/timeColumn.css (hunk 1 above)
Comment (adhrinae) [reply apps/calendar/src/css/timegrid/timeColumn.css:65]: 요건 이전에 작성된 코드를 옮긴거긴한데 한번 찾아서 바꿔보겠습니다.

**Review by lja1018 — APPROVED** [review]
리뷰 완료합니다.

File: apps/calendar/src/components/timeGrid/timezoneCollapseButton.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    'ic-arrow-left': !isCollapsed,
+  });
+
+  return (
+    <button
+      className={cls(addTimeGridPrefix('timezone-collapse-button'))}
+      aria-expanded={!isCollapsed}
+      onClick={() => eventBus.fire('clickTimezonesCollapseBtn', isCollapsed)}
+    >
+      <span className={iconClassName} role="img" />
```
[the remark this answers is not in the document]
Comment (adhrinae) [reply apps/calendar/src/components/timeGrid/timezoneCollapseButton.tsx:21]: 저도 썩 내키진 않지만 아래의 문서를 참고했어요.
더 제대로 하고자 한다면 `aria-label` 도 넣긴 해야하지요.

**Pull request merged**